crates/workspace/src/pane.rs

```diff
@@ -1119,6 +1119,19 @@ impl Pane {
         )
     }
 
+    fn copy_path(&mut self, _: &CopyPath, path: PathBuf, cx: &mut ViewContext<Self>) {
```

Comment:
These two methods are useless now, they do the exact same oneliner thing with `cx.write_to_clipboard(ClipboardItem::new(path.to_string_lossy().to_string()));`.

They accept different arguments for some reason, but that changes nothing.
Let's either remove both methods entirely or get the abs path inside one of them.

Reply:
I was trying to get absolute path out of the entry and didn't notice they did the same thing… I will move that line to the call site and remove the functions then